src/FFmpegReader.cpp

```diff
@@ -595,7 +595,8 @@ void FFmpegReader::Close() {
 		// Drain any packets from the decoder
 		packet = NULL;
 		int attempts = 0;
-		while (packets_decoded < packets_read && attempts < 256) {
+		int max_attempts = 128;
+		while (packets_decoded < packets_read && attempts < max_attempts) {
```

Comment:
Typically this loop never approaches the `max_attempts`, and instead breaks out of the loop as soon as the requested frame is found, typically within a few iterations. However, there are rare cases where an infinite loop can arise, and we don't want this to loop forever.